### PR TITLE
Adding Word limit to the input fields [Enhancement]

### LINKED ISSUE #190: [Enhancement] giving minimum word limit to ever thing
Description
There is no limit on the text currently and to make it more optimized we need to limit the text.

I am adding letter limit to post title, post description, comment, organization description, organization attendees description, event title, event description.

Issue Severity
Moderate

Observed Behavior:
No limit on the text 

Expected Behavior:
There should be limit on the text size

### PR DESCRIPTION
**Pull request by yasharth291**
**What kind of change does this PR introduce?**

[Enhancement] It adds limit to the text fields and also validates them

**If relevant, did you update the documentation?**

NO

**Other information**
It was important to reduce the server load to limit the words user is putting inside the text field.

Fixes #190

### CODE REVIEW

**Comment by yasharth291** [comment]
@Sagar2366 this pr is solved

File: lib/views/pages/newsfeed/addPost.dart [hunk 1]
```diff
@@ -93,17 +101,25 @@ class _AddPostState extends State<AddPost> {
...
+              child: Container(
               child: TextFormField(
+                maxLines: null,
+                inputFormatters: [
+                  LengthLimitingTextInputFormatter(10000)
+                ],
+                keyboardType: TextInputType.multiline,
                 controller: textController,
                 validator: (String value) {
+                  if (value.length > 10000) {
```
Comment (Sagar2366) [inline lib/views/pages/newsfeed/addPost.dart:116]: reduce title size to 30

**Review by Sagar2366 — APPROVED** [review]

**Comment by Sagar2366** [comment]
@yasharth291 resolve merge conflicts on this PR.

**Comment by yasharth291** [comment]
resolved sir

**Comment by yasharth291** [comment]
@Sagar2366 can it be merged now

**Pull request merged**